.github/workflows/otp_pkg.yaml

```diff
@@ -14,7 +27,14 @@ jobs:
       - name: Matrix Definition
         id: matrix-def
         run: |
-           echo "otp-version=${{ github.event.client_payload.otp }}" >> "$GITHUB_OUTPUT"
+          if [ "${{ github.event_name }}" = "repository_dispatch" ]; then
+            echo "otp-version=\"${{ github.event.client_payload.otp }}\"" >> "$GITHUB_OUTPUT"
+          elif [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+            echo "otp-version=\"${{ inputs.otp_version }}\"" >> "$GITHUB_OUTPUT"
+          else
+            echo "otp-version=\"27.2.3\"" >> "$GITHUB_OUTPUT"
```

Comment:
Same here I would add a comment, that such default is just used for developing the script